cachi2/core/package_managers/rpm/main.py

```diff
@@ -359,11 +369,20 @@ def raise_exception(message: str) -> None:
                 raise_exception(f"Unmatched checksum of '{file_path}' != '{digest}'")
 
 
+def _is_rpm_file(file_path: Path):
+    """Check if it's a rpm file."""
+    if file_path.suffix == ".rpm":
```

Comment:
This is a good helper method! It could be made even simpler:
```suggestion
return file_path.suffix == ".rpm"
```